content/_code-samples/escrow/js/cancel-escrow.js

```diff
@@ -1,40 +1,39 @@
 'use strict'
-const RippleAPI = require('ripple-lib').RippleAPI
+if (typeof module !== "undefined") {
+  // Use var here because const/let are block-scoped to the if statement.
+  var xrpl = require('xrpl')
+}
 
-const myAddr = 'rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn'
-const mySecret = 's████████████████████████████'
+const main = async () => {
+  try {
+    // Connect -------------------------------------------------------------------
+    const client = new xrpl.Client('wss://s.altnet.rippletest.net:51233');
+    await client.connect();
+    
+    // Get credentials from the Testnet Faucet ------------------------------------
+    console.log("Requesting an address from the Testnet faucet...");
+    const { wallet } = await client.fundWallet();
+    console.log("Wallet: ", wallet.address);
 
-const myEscrowCancellation = {
-  owner: myAddr,
-  escrowSequence: 366
-}
-const myInstructions = {
-  maxLedgerVersionOffset: 5
-}
+    const escrowCancelTransaction = {
+      "Account": wallet.address,
+      "TransactionType": "EscrowCancel",
+      "Owner": wallet.address,
+      "OfferSequence": 34821001, // Sequence number
+    };
+
+    xrpl.validate(escrowCancelTransaction);
 
-const api = new RippleAPI({server: 'wss://s2.ripple.com'})
+    // Sign and submit the transaction --------------------------------------------
+    console.log('Signing and submitting the transaction:', JSON.stringify(escrowCancelTransaction, null,  "\t"));
+    const response  = await client.submitAndWait(escrowCancelTransaction, { wallet });
+    console.log(`Finished submitting! ${JSON.stringify(response.result, null, "\t")}`);
```

Comment:
nit: It's a little hard to see the 'Finished submitting!' message amongst the large output. Adding an extra line should help it be easier to see.
```suggestion
    console.log(`Finished submitting! \n${JSON.stringify(response.result, null, "\t")}`);
```

Reply:
Added